### ISSUE TITLE
Can we provide a better review assignment email subject.

### ISSUE THREAD

**Issue by rjsparks** [ietf-tools/datatracker#4684]
### Discussed in https://github.com/ietf-tools/datatracker/discussions/3760

<div type='discussions-op-text'>

<sup>Originally posted by **rjsparks** March 29, 2022</sup>
See the thread starting at https://mailarchive.ietf.org/arch/msg/tools-discuss/MeFTTsnSBu4kaT3Xx5c9Jj_UXws.

There is a proposal, gaining support, to change the review assignment email subject line to
```
<team name> review assignment of <draft-foo> by <deadline>
```</div>

**Comment by rjsparks** [comment]
Draft PR still at #4769 - can we get that ready to merge this sprint?

**Comment by rjsparks** [comment]
closed with #5415
